docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -400,49 +400,67 @@ The following table lists the measurement data that the gyroscope rotation vecto
 
 The Heart Rate Monitor (HRM) LED green sensor measures the amount of green light that is reflected back from a person's blood vessel.
 
-The following table lists the measurement data that the HRM LED green sensor provides.
+The following table lists the measurement data that the HRM LED green sensor provides:
 
 **Table: Measurement data detected by the HRM LED green sensor**
 
 | Measurement                      | Type                 | Range                                   | Unit         |
 |----------------------------------|----------------------|-----------------------------------------|--------------|
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
-| values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 1081216 | -            |
+| values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
+
+<a name="hrm_green_batch"></a>
+## Heart Rate Monitor LED Green batch Sensor
+
+The Heart Rate Monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor must be power-efficient and work in a batch manner.
```

Comment:
As a conclusion on our discussion https://github.com/Samsung/tizen-docs/pull/1083#discussion_r411830136 , we suggested the following update: 
```suggestion
The Heart Rate Monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor must be power-efficient and support batch processing.
```
@bbrto21 if you think this update doesn't change the meaning,
would you please apply this update?

Reply:
@nawab-samsung Thank you for suggestion, I accept your opinion and updated it